lua/mediaplayer/services/base/init.lua

```diff
@@ -26,7 +26,7 @@ end
 
 local HttpHeaders = {
 	["Cache-Control"] = "no-cache",
-	["Connection"] = "keep-alive",
+	--["Connection"] = "keep-alive",
```

Comment:
Did anyone figure out why this was causing the issue with HTTP requests? It never has the `unsuccessful` error on my end 🤔  It is only on dedicated servers?